packages/tcc/build.sh

```diff
@@ -1,20 +1,23 @@
+_COMMIT=4e91d38ddcdb659a4b2c298850c6ccf115073748
+_COMMIT_DATE=20240225
+_COMMIT_SHA256=26974776737d9da876395cc72a005a63526e305aeaa28ebf84c56778a8d9fdac
+
 TERMUX_PKG_HOMEPAGE=https://bellard.org/tcc/
 TERMUX_PKG_DESCRIPTION="Tiny C Compiler"
 TERMUX_PKG_LICENSE="LGPL-2.1"
 TERMUX_PKG_MAINTAINER="@termux"
-_COMMIT=6a24b762d3e1086dcffd002c68cb5ca3a33a5c6d
-_COMMIT_DATE=20230415
-TERMUX_PKG_VERSION=1:0.9.27-p${_COMMIT_DATE}
-TERMUX_PKG_REVISION=1
+TERMUX_PKG_VERSION=1:0.9.28-p${_COMMIT_DATE}
 TERMUX_PKG_SRCURL=git+https://repo.or.cz/tinycc.git
-TERMUX_PKG_SHA256=467792219d0172f594ec71bcd6bac9dbb25308cbe9f708bab455b717148b491b
+TERMUX_PKG_SHA256=$_COMMIT_SHA256
```

Comment:
```suggestion
TERMUX_PKG_SHA256=26974776737d9da876395cc72a005a63526e305aeaa28ebf84c56778a8d9fdac
```
Since `$_COMMIT_SHA256` is only ever used as the value of `$TERMUX_PKG_SHA256`,
why have it in the first place?